tests/unittest/scripts/test_cbts_coverage_pilot.py

```
@@ -0,0 +1,169 @@
+PILOT_PATH = REPO_ROOT / "jenkins" / "scripts" / "cbts" / "coverage_pilot.py"
+PR_API_URL = "https://api.github.com/repos/NVIDIA/TensorRT-LLM/pulls/123"
+TEST_PILOT_USERS = frozenset({"pilot-user"})
+JSONValue: TypeAlias = Union[
+    None,
+    bool,
+    int,
+    float,
+    str,
+    list["JSONValue"],
+    dict[str, "JSONValue"],
+]
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)